Engines/Wine/Verbs/msxml3/script.js

```diff
@@ -0,0 +1,45 @@
+include("engines.wine.engine.object");
+include("engines.wine.plugins.override_dll");
+include("utils.functions.net.resource");
+include("utils.functions.filesystem.files");
+
+
+/**
+* Verb to install msxml3
+* @returns {Wine} Wine object
+*/
+Wine.prototype.msxml3 = function () {
+
+    var setupFile32 = new Resource()
+        .wizard(this.wizard())
+        .url("https://media.codeweavers.com/pub/other/msxml3.msi")
+        .checksum("d4c2178dfb807e1a0267fce0fd06b8d51106d913")
+        .name("msxml3.msi")
+        .get();
+
+    remove(this.system32directory() + "/msxml3.dll")
+    this.overrideDLL()
+        .set("native", ["msxml3"])
+        .do();
+
+    this.wizard().wait(tr("Please wait while {0} is installed...", "msxml3"));
+    this.run(setupFile32, ["/q:a", "/c:msxml3.msi /q"], null, false, true);
```

Comment:
This does not looks like winetricks one : w_try "$WINE" msiexec /i msxml3.msi /

Reply:
We don't have a separate command for msi files. But the installer works fine.

Reply:
Just for clarification: `wine.run` will behave correctly if it detects a msi (depending on the file suffix).